client/src/components/layout/UserBuildsTile.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const UserBuildTile = ({ build }) => {
+    const { id } = build
```

Comment:
if you are destructuring the id you should also include the title - `{ id, title }` to be consistent